scripts/undeploy.sh

```diff
@@ -66,6 +66,11 @@ function remove_cluster_objects
             kubectl delete $obj $(kubectl get $obj | grep '^verticadb-operator-' | cut -d' ' -f1) || true
         fi
     done
+    # Remove CRD
+    if kubectl get crd | grep '^vertica'
+    then
+        kubectl delete crd $(kubectl get crd | grep '^vertica' | cut -d' ' -f1) || true
```

Comment:
You need to delete crd "eventtriggers.vertica.com" as well. You can grep crds that end with '.vertica.com'

Reply:
Sure, updated in https://github.com/vertica/vertica-kubernetes/pull/988/commits/6df4b04efa61b8cddec8fcae8e928f090dd108a7